config/utils.go

```diff
@@ -86,29 +86,21 @@ func desanitize(actual, target *model.Config) {
 	}
 }
 
-// fixConfig patches invalid or missing data in the configuration, returning true if changed.
-func fixConfig(cfg *model.Config) bool {
-	changed := false
-
+// fixConfig patches invalid or missing data in the configuration.
+func fixConfig(cfg *model.Config) {
```

Comment:
@streamer45 - Just to confirm, the `changed` result is not required at the call site right? I can see that it's not used, but should it be?

Reply:
I am guessing it was needed originally but not since the last major change (feature flags). I am okay with removing it for now. All this code is getting reviewed soon anyway and if needed it's easy to add it back.